test/evp_test.c

```diff
@@ -66,9 +66,9 @@ struct evp_test_method_st {
  * Linked list of named keys.
  */
 typedef struct key_list_st {
+    struct key_list_st *next;
     char *name;
     EVP_PKEY *key;
-    struct key_list_st *next;
```

Comment:
Ok...  it's just a bit confusing to see these unrelated changes being "sneaked in"